src/monitor-plan-workspace/monitor-plan-checks.service.ts

```diff
@@ -48,6 +50,23 @@ export class MonitorPlanChecksService {
             }),
           );
         });
+
+        monitorLocation.unitControls?.forEach(unitControl => {
+          promises.push(
+            new Promise(async (resolve, _reject) => {
+              const results = this.unitControlsChecksService.runChecks(
+                unitControl,
+                monitorLocation.unitRecordId,
+                monitorLocation.id,
```

Comment:
```suggestion
                monitorLocation.unitRecordId,
                monitorLocation.id,
```

These wont be provided by user.